cmd/fs-v1-helpers.go

```diff
@@ -225,6 +225,15 @@ func fsStatDir(ctx context.Context, statDir string) (os.FileInfo, error) {
 	return fi, nil
 }
 
+// Returns if the dirPath is a directory.
+func fsIsDir(ctx context.Context, dirPath string) bool {
```

Comment:
To avoid little code duplication, we can do this:
```golang
func fsIsDir(ctx context.Context, dirPath string) bool {
  fi, err := fsStatDir(ctx, dirPath)
  return err == nil
}
```